mlir/include/mlir/Dialect/MIOpen/Generator/Conv2dGenerator.h

```diff
@@ -78,8 +78,19 @@ class Conv2dGenerator {
   static Optional<miopen::ConvOpType> getOpForName(const StringRef name);
   static const char *getNameForOp(const miopen::ConvOpType);
 
+  static inline constexpr int64_t outputDim(int64_t inputLen, int64_t filLen,
+                                            int64_t leftPadLen,
+                                            int64_t rightPadLen,
+                                            int64_t strideLen, int64_t dilLen) {
+    return (inputLen + leftPadLen + rightPadLen - (filLen - 1) * dilLen - 1) /
+               strideLen +
+           1;
```

Comment:
Formatting looks a bit off.

Reply:
*shrugs* - that's how it was in mlir-miopen-driver.cpp